extension/goext/auth.go

```diff
@@ -21,6 +21,10 @@ type IAuth interface {
 	HasRole(context Context, role string) bool
 	// GetTenantName return name from the given context
 	GetTenantName(context Context) string
+	// GetTenantID return id of the tenant from the given context
+	GetTenantID(context Context) string
```

Comment:
Both `tenant_id` and `domain_id` (since [[CW-1339] add domain id to context](https://github.com/cloudwan/gohan/pull/744)) are included in context, what is the point of that functions then? Aren't they redundant?